ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/FastWorldStateDownloader.java

```diff
@@ -107,6 +107,16 @@ private IntSupplier downloadStateValue(final Function<FastWorldDownloadState, In
         return failed;
       }
 
+      Optional<BlockHeader> checkNull = Optional.ofNullable(fastSyncState.getPivotBlockHeader().get());
+      if(checkNull.isEmpty()){
+        LOG.error("Pivot Block not present");
+        final CompletableFuture<Void> failed = new CompletableFuture<>();
+        failed.completeExceptionally(
```

Comment:
this completeExceptionally looks right to me

Reply:
Ok, I'll check it.
Thanks